cves/CVE-2015-8662.yml

```diff
@@ -342,7 +383,14 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Reading the initial commit of the jpeg2000dwt.c file, it the initial 
+    functionality was created based on an already existing jpeg2000
+    decoding/encoding library created during Google Summer of Code. Potentially
+    this issue was present in that library and remained when brought over to
+    FFmpeg. Additionally, after the initial commit the file was commited to 
+    by a single developer, never having another set of eyes working on the file
+    could have led to the vulnerability lying dormant for as long as it did. 
```

Comment:
So are you saying that whoever decided to use this code from the already existing jpeg200 library should better tried to understand the code and looked it over better when they decided to use it to see if there were any issues with it?